index.html

```diff
@@ -0,0 +1,50 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="preconnect" href="https://fonts.googleapis.com" />
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
+    <link rel="preconnect" href="https://fonts.googleapis.com" />
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Crete+Round&family=Harmattan&family=Inter:wght@100&family=Poppins:wght@200&family=Roboto+Serif:wght@100&family=Roboto:ital,wght@0,400;1,100&display=swap"
+      rel="stylesheet"
+    />
+
+    <link rel="stylesheet" href="CSS/style.css" />
+    <title>My-Portfolio</title>
+  </head>
+  <body>
+    <section>
+      <header>
+        <nav>
+          <div class="mobile-nav">
+            <a href="#" class="name_logo">Gidiz Logo</a>
+            <img class="menu-bar" src="icons/menu.png" alt="Menu" />
+          </div>
+        </nav>
+      </header>
+
+      <div class="container">
+        <h1>Hey There. I’m Gideon</h1>
+        <h2>I’am a Software Developer</h2>
+        <p>
+          I can help you build a product , feature or website Look through some
+          of my work and experience! If you like what you see and have a project
+          you need coded, don’t hestiate to contact me.
+        </p>
+
+        <div class="social">
+          <img src="icons/social/github.png" alt="github" />
+          <img src="icons/social/linkedin.png" alt="linkedin" />
+          <img src="icons/social/angellist.png" alt="angellist" />
+          <img src="icons/social/twitter.png" alt="twitter" />
+          <img src="icons/social/medium.png" alt="medium" />
+        </div>
+        <img class="side_img" src="icons/side_img.png" alt="" />
```

Comment:
[Required] Please fix the background image here it should look the same as the [template](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=1%3A1471)
![image](https://user-images.githubusercontent.com/74744787/158691272-0ce102e8-edd4-4f70-a5f7-6c0eac71938f.png)

![image](https://user-images.githubusercontent.com/74744787/158691347-00f74e09-cc35-4a41-a74e-dea70b3ce8bd.png)